costmap_2d/test/inflation_tests.cpp

```diff
@@ -97,6 +97,10 @@ void validatePointInflation(unsigned int mx, unsigned int my, Costmap2D* costmap
           continue;
         }
 
+        if (dist == bin->first) {
+          dist += 0.001;
+        }
+
```

Comment:
FYI: This change was already applied to the rr-devel branch through [this PR]( https://github.com/rapyuta-robotics/ros-navigation/pull/36/files#diff-199dfeab85a5f3716cd0e86414c16e813e330bbf98fdaa029cf8790631cc7a4aR106)

(How does git display these changes? shouldn't there be a merge conflict here?)

Reply:
wow, this PR is old. I was not tracking it, thanks.
I will check if we need it yet.

> (How does git display these changes? shouldn't there be a merge conflict here?)

There is a merge conflict. And a "Resolve conflicts" button.